client/my-sites/customer-home/locations/tertiary/manage-site/index.jsx

```diff
@@ -63,4 +63,22 @@ const mapStateToProps = ( state ) => {
 	};
 };
 
-export default connect( mapStateToProps )( ManageSite );
+const mapDispatchToProps = ( dispatch ) => ( {
+	renderCard: ( card ) =>
+		cardComponents[ card ] &&
+		dispatch(
+			withAnalytics(
+				composeAnalytics(
+					recordTracksEvent( 'calypso_customer_home_card_impression', {
+						card,
+					} ),
+					bumpStat( 'calypso_customer_home_card_impression', card )
+				),
+				React.createElement( cardComponents[ card ], {
+					key: card,
+				} )
```

Comment:
Oh hmm not sure how this was working before, normally the withAnalytics decorates a redux action. I can put up an alternative attempt